README.md

```diff
@@ -23,14 +23,23 @@ A minimalist Vim plugin manager.
 [startup-time]: http://junegunn.kr/images/vim-startup-time.png
 
 ### Installation
-
 [Download plug.vim](https://raw.githubusercontent.com/junegunn/vim-plug/master/plug.vim)
+###### Unix
 and put it in ~/.vim/autoload
```

Comment:
Ok, then:
"Download plug.vim and put it in `~/.vim/autoload` (UNIX) or `~/vimfiles/autoload` (Windows).